src/openfl/_internal/renderer/canvas/CanvasGraphics.hx

```diff
@@ -1253,6 +1253,10 @@ class CanvasGraphics {
 			
 			var width = graphics.__width;
 			var height = graphics.__height;
+
+			if (graphics.__bitmap != null) {
+				graphics.__bitmap.disposeImage ();
+			}
 			
```

Comment:
In the following lines after this call, we check if our graphics are visible. If not, we flag our bitmap for disposal.

If we will render, we determine the size. If we can re-use our previous bitmap data, we do. If not, we allocate a new one. Calling `disposeImage` here will work against these optimizations

Reply:
Not sure I understand, in the following lines we either set the `__bitmap` to `null` or recreate it with `BitmapData.fromCanvas`, so it always makes sense to release the resources of the previous bitmap?